src/components/Banner/Banner.scss

```diff
@@ -131,10 +131,6 @@ $secondary-action-horizontal-padding: 1.5 * spacing(tight);
     $newDesignLanguageOffset: $newDesignLanguageSpacing - spacing();
     padding: spacing() spacing() $newDesignLanguageSpacing;
 
-    &.hasDismiss {
-      padding-right: calc(#{2 * spacing()} + var(--p-icon-size));
-    }
```

Comment:
Removing from under `.withinContentContainer` since it wasn't being applied to `.withinPage`